docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+services:
+  db:
+    image: postgres:14.15-alpine3.21
+    restart: always
+    env_file:
+      - .env
+    volumes:
+      - postgres_data:/var/lib/postgresql/data
+
+  app:
+    build: .
+    command: >
+      sh -c "python manage.py wait_for_db &&
+            python manage.py migrate &&
+            python manage.py runserver 0.0.0.0:8000"
```

Comment:
The command to run the Django application includes `python manage.py runserver`, which is suitable for development. For production, consider using a production-ready server like Gunicorn.